src/VisualStudio/Core/Def/Implementation/Workspace/DetailedErrorInfoDialog.xaml.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Forms;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using Microsoft.VisualStudio.PlatformUI;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation
+{
+    /// <summary>
+    /// Interaction logic for UserControl1.xaml
+    /// </summary>
+    public partial class DetailedErrorInfoDialog : DialogWindow
+    {
+        string errorInfo;
+
+        internal DetailedErrorInfoDialog(string title,  string errorInfo)
+        {
+            InitializeComponent();
+            this.errorInfo = errorInfo;
+            this.Title = title;
+            stackTraceText.AppendText(errorInfo);
+            this.CopyButton.Content = ServicesVSResources.Copy_to_clipboard;
+            this.CloseButton.Content = ServicesVSResources.Close;
+        }
+
+        private void CopyMessageToClipBoard(object sender, RoutedEventArgs e)
+        {
+            System.Windows.Clipboard.SetText(errorInfo);
```

Comment:
This throws in weird ways (e.g. rdpclip.exe not running in a TS session) - consider catching exceptions.